src/query.rs

```diff
@@ -215,7 +190,7 @@ mod tests {
         assert_eq!(expected_paths, actual_paths);
     }
 
-    fn check_query_execution(query_name: &str) {
+    pub fn check_query_execution(query_name: &str) {
```

Comment:
Can this be `pub(in query)` or something narrower like that, rather than `pub`?

Reply:
Oh, good idea. I didn't think it's possible to narrow the scope like that in Rust. I'll change it. I think it's not possible to narrow it further, because I'm calling this function from another (sibling) module, so it's not possible to narrow it to a module and I guess the next wider scope is per-file. But idk, I still have to read about the possible scope narrowing in Rust.

Reply:
I think that function is in `mod query -> mod tests` and is used from `mod query` and from `mod query -> mod tests_lints`. So if my mental model of this is correct, narrowing to `pub(in query)` should work since all child modules of `query` should be able to access it too.

I don't think there's real "per file" scoping in Rust, just per module with files as implicit modules.

Reply:
Perfect!